### ISSUE TITLE
Define spaces and indexes on storages using provided DB schema

### ISSUE THREAD

**Issue by Totktonada** [tarantool/graphql.0#233]
- Implement converter from DB schema to space/index formats.
  - It worth to convert from business model format (after #144).
  - It worth to implement this converter as a separate module to alow reusing in other projects.
- Provide an API via `graphql.storage`.

This will allow to avoid defining of a database schema twice in different formats.

**Comment by Totktonada** [comment]
I'm going to archive the repository. I'll proceed as follows:

* Mark all open pull requests with the `archived` label and close.
* Mark all open issues with `archived` and `wontfix` labels and close.
* Archive the repository.

Consider the following projects:

* https://github.com/tarantool/graphql
* https://github.com/tarantool/graphqlapi
* https://github.com/tarantool/graphqlide
